### ISSUE TITLE
Release 1.7.8.6

### ISSUE THREAD

**Issue by matks** [PrestaShop/PrestaShop#27973]
This issue tracks the status of the PrestaShop 1.7.8.6 release

### Information

Q                                     | A
-----------------------|---------------
**Type**                        | Patch version
**Build date**                |  07-05-2022
**Milestone**                | [1.7.8.6](https://github.com/PrestaShop/PrestaShop/milestone/82)
**Kanban**                    | [1.7.8.6](https://github.com/PrestaShop/PrestaShop/projects/25)
**Delivery date**   |  16-05-2022
**Release manager tech**           |  @matks 
**Release manager product**     |  @eshraw 

### Current status

- [x] Development
- [x] Build
- [x] QA validation
- [x] Release
- [x] Available for upgrade
- [x] Docker image

**Comment by matks** [comment]
### Release issue reported

I'm checking whether everything's OK for 1.7.8.6 release to be built and tested next week. I found a problem.

Core Pull Request https://github.com/PrestaShop/PrestaShop/pull/27162 introduced a SQL structure change.
Autoupgrade Pull Request https://github.com/PrestaShop/autoupgrade/pull/455 was submitted to add SQL migration script inside autoupgrade module.

BUT the autoupgrade module uses its own SQL files since v5, and v5 is not released yet.

If I am not wrong, if people want to use the current autoupgrade module 4.14.0 to upgrade from 1.7.8.5 to 1.7.8.6, we need to put the SQL script inside https://github.com/PrestaShop/PrestaShop/tree/1.7.8.x/install-dev/upgrade/sql like the previous 1.7.8.x releases

==> submitted in bug report https://github.com/PrestaShop/PrestaShop/issues/28272

**Comment by matks** [comment]
### Status update

1.7.8.6 should be built on Tuesday 19th of April BUT there are 2 blocking issues
- https://github.com/PrestaShop/PrestaShop/issues/28272 : this will be an issue for upgrading from 1.7.x versions to 1.7.8.6
- https://github.com/PrestaShop/PrestaShop/issues/28264 : blockreassurance 5.1.1 is included in 1.7.8.6 so this major bug would be deployed

**Comment by matks** [comment]
## Status update

Two issues left before being able to build 1.7.8.6

- https://github.com/PrestaShop/PrestaShop/issues/28264 -> I can reproduce now
- https://github.com/PrestaShop/PrestaShop/issues/28299

Also a new release of upgrade module will be needed for smooth 1.7.x.y -> 1.7.8.6 upgrades

**Comment by matks** [comment]
## Status update

One issue left before being able to build 1.7.8.6

## List of items to deliver before being able to build a 1.7.8.6 release

### Fix blockreassurance module
- [ ] https://github.com/PrestaShop/PrestaShop/issues/28264 bugfix submitted
- [ ] Release new blockreassurance version
- [ ] Include blockreassurance into the Core

### Release a new autoupgrade module
- [ ] Release a new version of autoupgrade module https://github.com/PrestaShop/autoupgrade/pull/481

A new release of upgrade module will be needed for smooth 1.7.x.y -> 1.7.8.6 upgrades

**Comment by matks** [comment]
## Status update

Both issues above proved to be far more complex and severe than expected. In order not to postpone 1.7.8.6 any later, I reduced the scope of the release to the very items needed. 1.7.8.6 core release should not be postponed because of issues inside the upgrade module or the blockreassurance module, these should be fixed and released on their own.

## List of items to deliver before being able to build a 1.7.8.6 release

### Blockreassurance module
- [x] Downgrade blockreassurance version to 5.1.0 in the core https://github.com/PrestaShop/PrestaShop/pull/28362

### Release a new autoupgrade module
- [ ] Release a new version of autoupgrade module https://github.com/PrestaShop/autoupgrade/pull/483

**Comment by matks** [comment]
## Status update

One more blocking issue https://github.com/PrestaShop/PrestaShop/issues/28372

**Comment by matks** [comment]
## Status update

The test of [autoupgrade 4.14.1](https://github.com/PrestaShop/autoupgrade/pull/483) reported issues for 1.7.7.8 -> 1.7.8.6 upgrade. We need to investigate if these are because of 1.7.8.x content, then it would be regressions that block the 1.7.8.6 release.

**Comment by matks** [comment]
## Status update

The autoupgrade issue was not confirmed so we dismissed it. I generated a first 1.7.8.6 build (attached below) that will be entrusted to QA team for validation from Monday.

EDIT: GitHub does not want to me attach the ZIP 😫 pretending ZIP are not supported

**Comment by matks** [comment]
Please note that Core software upgrades 1.7[...] to 1.7.8.6 must be done using the [autoupgrade module 4.14.1](https://github.com/PrestaShop/autoupgrade/pull/483) that is also ready for QA before being released.

Both 1.7.8.6 Core version and autoupgrade 4.14.1 version should be validated and released together.

**Comment by khouloudbelguith** [comment]
Hello :slightly_smiling_face:

Final report of QA on the 1.7.8.6 build 1 :

- The automated test campaign on the build is done and OK.
- The manual functional test campaign is done and OK.
- The exploratory test session is now complete. The QA Team did not find any 1.7.8.6 regressions.

Summary of some issues detected during the test:

- https://github.com/PrestaShop/PrestaShop/issues/28478 Minor regression PS1785
- https://github.com/PrestaShop/PrestaShop/issues/28462 Trivial regression module searchbar
- https://github.com/PrestaShop/PrestaShop/issues/28492 Minor regression of the module searchbar
- https://github.com/PrestaShop/PrestaShop/issues/28476 Minor regression PS1783
- https://github.com/PrestaShop/PrestaShop/issues/28484 Minor regression PS1780
- An already found bug on 1.7 is a bit more complex : (see last comments on https://github.com/PrestaShop/PrestaShop/issues/15041#issuecomment-1125855989  )
- https://github.com/PrestaShop/PrestaShop/issues/28485
- https://github.com/PrestaShop/PrestaShop/issues/28470

Some other bugs (no 1.7.8 regressions) were found, but the issues are yet to be created.
In conclusion, it's a GO to continue the release! QA
Also, for the autoupgrade release https://github.com/PrestaShop/autoupgrade/pull/483, It is OK, we validate it.

Thank you

**Comment by matks** [comment]
As it's Friday, let's release on Monday 👍  thank you all the QA members!

**Comment by matks** [comment]
Release is in progress

1. Autoupgrade module 4.14.1 released on GitHub
2. PrestaShop 1.7.8.6 released on GitHub

**Comment by matks** [comment]
Release note is ready, I will complete the release process tomorrow at 9AM

https://github.com/PrestaShop/prestashop.github.io/pull/1081

(don't deploy in the evening 😋 )

**Comment by matks** [comment]
## Status update

1. PrestaShop 1.7.8.6 released on prestashop.com
2. Release note published https://build.prestashop.com/news/prestashop-1-7-8-6-maintenance-release/

**Comment by matks** [comment]
Available for upgrade

**Comment by Murat-06** [comment]
I am using version 1.7.8.6 and the products I sell are single stock products, when the customer adds the product to the cart, the product is out of stock label appears and the customer leaves the site because there is no product left. It is not like this in Presta 1.6 version when I make it. When you add it to the cart, the number of products does not decrease and it does not give an out of stock warning. When I check other platforms, it doesn't give an out of stock warning saying your product has been added to the cart. I hope this problem can be resolved with a new update in version 1.7.8.6 Thank you

**Comment by matks** [comment]
Hi @Murat-06 I think you should report this as a bug https://github.com/PrestaShop/PrestaShop/issues/new?assignees=&labels=Bug%2CNew&template=1_bug_report.yml

but it does not look like a known bug and it's really weird

this bug is so bad I don't think we would have missed it when testing I think the problem might come from something else than the software

**Comment by matks** [comment]
Docker images are available https://hub.docker.com/r/prestashop/prestashop/
